src/Microsoft.Extensions.FileSystemGlobbing/Abstractions/FileInfoWrapper.cs

```diff
@@ -5,28 +5,35 @@
 
 namespace Microsoft.Extensions.FileSystemGlobbing.Abstractions
 {
+    /// <summary>
+    /// Wraps an instance of <see cref="System.IO.FileInfo" /> to provide implementation of <see cref="FileInfoBase" />
+    /// </summary>
     public class FileInfoWrapper : FileInfoBase
     {
-        private FileInfo _fileInfo;
+        private readonly FileInfo _fileInfo;
 
+        /// <summary>
+        /// Initializes instance of <see cref="FileInfoWrapper" /> to wrap the specified object <see cref="System.IO.FileInfo" />
```

Comment:
Not required. But it made it easier to read since "FileInfo" is such an overloaded term. FileInfo, FileInfoBase, FileSystemInfo, FileInfoWrapper...etc